Automattic/wp-calypso
# ToS acceptance tracking: Update the file upload endpoint

#### Changes proposed in this Pull Request

* Update the file upload endpoint used for the screenshots zip file. The original PR that introduced the screenshot feature is in #57741. 
* This change is as per the advise given in pMz3w-eJp#comment-96860 to reuse the existing file upload endpoint.

#### Testing instructions



* Check out this branch to your local instance
* In test/e2e folder of your local Calypso folder, run yarn install
* Apply D78318-code and follow the setup instructions given in the "Test Plan" of that patch.
* In the `test/e2e` folder, run `BABEL_ENV=test yarn jest specs/martech/tos-screenshots__checkout.ts` and also the other specs in the martech folder.
* Verify that fullpage screenshots of signup, login, and checkout are stored on the blog, and the ToS link is visible in all these pages.



Related to #
